deploy/examples/jobs-service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoJobsService
```

Comment:
Is this related to the issue?

Reply:
nope but it was missing. Can put it in another PR is needed.

Reply:
Sure, np.

Reply:
Actually we don't need this since we have: https://github.com/kiegroup/kogito-cloud-operator/blob/master/deploy/crds/app.kiegroup.org_v1alpha1_kogitojobsservice_cr.yaml

But since we have examples for all services, I don't see any problems having it.